### PR TITLE
add capi categories

### PR DESCRIPTION
**Pull request by mytunguyen**
**What this PR does / why we need it**:
Add capi categories so that we can do `kubectl get cluster-api`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm
 go 1.12
 
 require (
+	github.com/aws/aws-sdk-go v1.23.4 // indirect
```
Comment (chuckha) [inline go.mod]: I think this might be leftover from working on CAPA?

Comment (mytunguyen) [reply go.mod]: yup! that's not supposed to be there

**Pull request merged**